script/Arcueid/Day 3/03_00_ARC03_1.txt

```diff
@@ -171,22 +180,23 @@ If someone were watching, they may think it's a strange habit, but I like to avo
 [sha:6f02d463051c2ee1b1153893bd99a42a811e9aef]{
 -- Page 13, Offset 3978.
 -- 壁際に振り向くと、そこにはメイド服の少女が立っていた。
-When I turn to face the wall where the voice came from, there's a girl in a maid's uniform standing there.
+// Some context spirited
+Turning to face the source of the sound, I see a girl in a maid's uniform standing by the wall.
 }
 [sha:90f7c9910e00d2ef89163402194a689cf7e59ff2]{
 -- Page 13, Offset 3979.
 -- あまりに気配がないので気がつかなかったが、ずいぶん前から居てくれたようだ。
-She didn't give any hint of her presence so I didn't notice her, but it appears she's been there for some time.
+She's so unobtrusive that I didn't even notice her, but it seems she's been there for a while.
 }
 [sha:f78fd89a8a53ee0496821273cc5e9525f7076fcd]{
 -- Page 13, Offset 3980.
 -- 昨日と同じように、俺が寝過ごさないように起こしに来てくれたのだろう。
-Just like yesterday, she must have come to wake me up so that I wouldn't oversleep.
+She must have come to wake me so that I wouldn't oversleep, same as she did yesterday.
```

Comment:
I wouldn't oversleep as I had yesterday

Reply:
Hmm, I think that's a slightly different meaning. I don't really get a sense of him oversleeping from 2_1

Reply:
yep, my mistake